esti/tmp_file_util.go

```diff
@@ -0,0 +1,46 @@
+package esti
+
+import (
+	crand "crypto/rand"
+	"math"
+	"os"
+)
+
+const defaultChunkSize = 1024
+
+func MakeTmpFile(numBytes int) (*os.File, func() error, error) {
+	file, err := os.CreateTemp("/tmp", "test")
```

Comment:
Package testing has a specific routine for creating temporary files, and you should use it.  Using `/tmp` is generally a bad idea, because it ignores possible environment settings of _other_ directories to use (useful when my /tmp is too small), and because its name is not associated with the test.

Just use [testing.TempDir](https://pkg.go.dev/testing#B.TempDir); you won't even need to clean it up.